docs/user-guide/persistent-volumes.md

```diff
@@ -23,7 +23,15 @@ PVs are resources in the cluster.  PVCs are requests for those resources and als
 
 ### Provisioning
 
-A cluster administrator will create a number of PVs. They carry the details of the real storage which is available for use by cluster users.  They exist in the Kubernetes API and are available for consumption.
+#### Manual
+A cluster administrator manually creates a volume resource and a PV API objects that contains the details about that volume. Once the PV object is created, it is available to fulfill PVCs.
+
+#### Dynamic (experimental)
+Volume plugins that support dynamic volume provisioning (GCE PD, AWS EBS, and Cinder) can automatically create a volume resource and the PV API object that represents the details of that volume. Dynamic volume provisioning is triggered by the user creating a PVC with an annotation. Once the volume resource and the PV object representing it is created, the PVC that triggered the dynamic provisioning binds to the volume. 
+
+Dynamic volume provisioning is an experimental feature. In this experimental state, it determines which provisioner (GCE PD, AWS EBS, or Cinder) to invoke based on the `cloudprovider` it is running in: If running in GCE/GKE, the GCE PD provisioner is used. If running in AWS, the EBS provisioner is used. If running in OpenStack, the Cinder provisioner is used. For all other cloud providers, or if `cloudprovider` is not specified, dynamic provisioning is not supported.
```

Comment:
For all other cloud providers, or if `cloudprovider` is not specified as dynamic provisioning is not supported.